src/rastervision/utils/files.py

```diff
@@ -56,6 +57,16 @@ def sync_dir(src_dir, dest_uri, delete=False):
     subprocess.run(command)
 
 
+def _is_raster(uri, s3_test=False):
+    if s3_test:
+        uri = uri.replace('s3://', '/vsis3/')
```

Comment:
I'm not sure why `s3_test` is a parameter. Shouldn't we always do this replacement?

Reply:
yes